ec_grasp_planner/data/object_param.yaml

```diff
@@ -75,8 +75,8 @@
         PISAHand: 1.0
         PISAGripper: 1.0
         ClashHand: 1.0
-      max: [ 0.14,  0.1]
```

Comment:
These changes were some fast and dirty hack. I don't recall why we did it, but we should not commit it to master. Please check the role of these values. I guess we use min and max to define a region inside the bin where we can grasp collision-free (collision with the bin's walls for the WAM robot)

Reply:
Yes they are used in the feasibility function to black list unreachable zones.. I guess we should not merge those adjusted values? Should I check with my example?